examples/advanced/main.tf

```diff
@@ -13,12 +13,12 @@ module "resource_group" {
 ##############################################################################
 # Create serviceID to use for resource key hmac
 #
-# NOTE: The module itself supports creating one, but this example shows how
-# you can use an existing one
+# NOTE: The module itself supports creating internally, but this example shows
+# how to use an existing ones
 ##############################################################################
-
-resource "ibm_iam_service_id" "resource_key_existing_serviceid" {
-  name        = "${var.prefix}-reskey-serviceid"
+resource "ibm_iam_service_id" "resource_keys_existing_serviceids" {
+  count       = 5
```

Comment:
to reduce the number of service IDs created in our account, can we just create 1 here re-use it?

Reply:
[No](https://github.com/terraform-ibm-modules/terraform-ibm-cos/actions/runs/8451636271/job/23150580839#step:7:3692)

```
Error: AdvancedExample 2024-03-27T12:30:38Z logger.go:66: │ Error: [ERROR] Error creating resource key: The service-id 4b6ed744-a2bd-4e94-a8c6-a929f6a56924 provided is in use by another application. For security reasons, each application must have it's own service-id.  Please submit the request with no service-id specified so a new service-id can be generated. with resp code: {
```

Each resource key requires a service id. They will be created as needed automatically if they are not specified.